endstate_correction/protocol.py

```diff
@@ -14,10 +14,10 @@
 class BSSProtocol:
     """This is a dataclass mimicking the BioSimSpace.Protocol."""
 
-    timestep: int  # fs
-    runtime: float  # ns
-    temperature: float  # K
-    pressure: float  # atm
+    timestep: unit.Quantity  # fs
```

Comment:
I think we could have both, where I could make a class that accepts both `unit.Quantity` or float as input. If a quantity is accepted it is converted to the right unit and stored as `unit.Quantity`. If the input is a float, the unit will be added to it and stored as `unit.Quantity`.

Reply:
that approach would mimic the standard approach `openMM` is using.
I think that would be a good compromise --- this would mean that anyone who wants to play it safe can add quantities explicitly, and experts that know the quantity can go ahead and provide the raw unitless input.
I'd still opt for timestep to be a `int` though.